src/redux/validation/validation.slice.ts

```diff
@@ -155,6 +155,9 @@ export const validationSlice = createSlice({
     },
   },
   extraReducers: builder => {
+    builder.addCase(setRootFolder.pending, state => {
+      state.lastResponse = undefined;
```

Comment:
what if the `setRootFolder` action fails? will we end up with the previous project with no `lastResponse`? or no project at all?